styles/components/confirmNewsletter.module.css

```
@@ -0,0 +1,227 @@
+    display: flex;
+    height: 56px;
+    justify-content: flex-end;
+    align-items: center;
+    gap: 12px;
+    align-self: stretch;
+    margin-bottom: 1.5rem;
+    background-color: var(--background);
+    border: 1px solid #cdcccc;
+    padding: 0 10px;
+    color: var(--brown);
+    background: white;
```

Biome (1.9.4)

[error] Unexpected shorthand property background after background-color



(lint/suspicious/noShorthandPropertyOverrides)